### PR TITLE
Get started with new S3 collection backend

### PR DESCRIPTION
**Pull request by FynnBe**

### CODE REVIEW

File: tests/test_scripts/test_utils/test_s3client.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    client = Client()
+    assert client.prefix.startswith("sandbox")
+    client.put_json("test/test1.json", "test")
+    client.put_json("test/dir/test2.json", "test")
+    client.put_json("test/dir/test3.json", "test")
+    assert set(client.ls("test/")) == {"test1.json", "dir"}
+    client.mv_dir("test/", "test_b/", bypass_governance_mode=BGM)
+    sleep(3)
+    assert set(client.ls("test/")) == {}
```
Comment (FynnBe) [inline tests/test_scripts/test_utils/test_s3client.py:18]: @jmetz this test fails here, any idea why removing objects from s3 does not work as I expect?

**Comment by jmetz** [comment]
@FynnBe : Ready for me to merge or you wanna keep working on some part of this?

File: tests/test_scripts/test_utils/test_s3client.py (hunk 1 above)
Comment (jmetz) [reply tests/test_scripts/test_utils/test_s3client.py]: Resolved by [4c1ed6a](https://github.com/bioimage-io/collection/pull/1/commits/4c1ed6a6bbcd5795d0105348fc2f769c077bad76)

**Review by jmetz — APPROVED** [review]
LGTM, and I know there's more planned in another PR IIUC

**Pull request merged**